go/vt/schemadiff/table.go

```diff
@@ -275,6 +277,46 @@ func (c *CreateTableEntity) diffTableCharset(
 	return ""
 }
 
+// isDefaultTableOptionValue sees if the value for a TableOption is also its default value
+func isDefaultTableOptionValue(option *sqlparser.TableOption) bool {
```

Comment:
Not for this PR, but should there be something similar for what the defaults are for column level options?

Reply:
for our discussion later on.